src/ai/backend/agent/server.py

```diff
@@ -827,15 +827,32 @@ async def server_main(
 @click.option(
     "--debug",
     is_flag=True,
-    help="Enable the debug mode and override the global log level to DEBUG.",
+    help="This option will soon change to --log-level TEXT option.",
+)
+@click.option(
+    "--log-level",
```

Comment:
FYI: You may narrow down the range of supported values using `Choices`.

https://click.palletsprojects.com/en/8.1.x/options/#choice-options